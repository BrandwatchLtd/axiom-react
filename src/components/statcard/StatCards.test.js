import React from 'react';
import renderer from 'react-test-renderer';
import { StatCards } from 'bw-axiom';

function getComponent(props = {}) {
  return renderer.create(
    <StatCards { ...props }>Lorem</StatCards>
  );
}

describe('StatCards', () => {
  it('renders with defaultProps', () => {
    const component = getComponent();
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
