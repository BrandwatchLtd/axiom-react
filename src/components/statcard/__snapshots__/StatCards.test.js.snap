// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StatCards renders with defaultProps 1`] = `
<div
  className="ax-grid__container ax-space--medium"
>
  <div
    className="ax-grid ax-grid--responsive ax-grid--gutters-horizontal--tiny ax-grid--gutters-vertical--tiny ax-grid--fill"
  >
    Lorem
  </div>
</div>
`;
